components/Forms/NumberField.tsx

```diff
@@ -65,6 +65,7 @@ export const NumberField: React.VFC<NumberFieldProps> = ({
         enterKeyHint="done"
         allowNegative={false}
         decimalSeparator={null}
+        aria-describedby={`help-text-enter-${name}`}
```

Comment:
I can't find `help-text-enter-xxx`. maybe you meant `helpText-xxx` ?

Reply:
<img width="885" alt="Screenshot 2023-03-14 at 1 32 51 PM" src="https://user-images.githubusercontent.com/9094315/225089893-570d6b12-17c3-425a-b81b-2213083cb067.png">